eng/tox/tox.ini

```diff
@@ -156,6 +158,8 @@ commands =
       -p {toxinidir} \
       -w {envtmpdir} \
       --package-type sdist
+    {envbindir}/python {toxinidir}/../../../eng/tox/create_whl_for_dependencies.py \
```

Comment:
I read through `create_package_and_install`. There is a part where it checks for the target already being installed, but I have changed my mind about ordering. I think you're g2g to leave this where it is.